contracts.go

```diff
@@ -46,20 +74,16 @@ func parseTemplate(b []byte) ContractTemplate {
 		} else if strings.HasPrefix(match, SIGN_PREFIX) {
 			signName := strings.TrimPrefix(match, SIGN_PREFIX)
 			contractTemplate.Signing = appendNew(contractTemplate.Signing, signName)
-		} else {
-			contractTemplate.Vars = appendNew(contractTemplate.Vars, match)
+		} else if strings.HasPrefix(match, VAR_PREFIX) {
+			varName := strings.TrimPrefix(match, VAR_PREFIX)
+			contractTemplate.Vars = appendNew(contractTemplate.Vars, varName)
 		}
 	}
 	return contractTemplate
 }
 
 // copy the template into a new dir and instantiate params file with empty values
-func newContract(cmd *cobra.Command, args []string) error {
-	if len(args) != 2 {
-		return fmt.Errorf("new expects two args: directory for new engagement and template path")
-	}
-
-	engagementPath, tmplPath := args[0], args[1]
+func newEngagement(engagementPath, tmplPath string) error {
```

Comment:
also not all contracts will be engagements. sometimes we just need to pass a signed board resolution for the board of directors. maybe there's other cases too